src/matrix/matrix-common.h

```diff
@@ -24,12 +24,13 @@
 // files in this directory.
 
 #include "base/kaldi-common.h"
-#include "matrix/kaldi-blas.h"
 
 namespace kaldi {
+// this enums equal to CblasTrans and SblasNoTrans constants from CBLAS library
+// we are writing them as numbers because we didn't include that library headers here  
```

Comment:
I'd rather say this: 'we are writing them as literals because we don't want to include "matrix/kaldi-blas.h", which puts many symbols things into global scope like "real" via the header "f2c.h". '

State why you the code does what it does, not what it does.